
sensor_msgs/PointCloud2 input_cloud

# Outlier Removal 
bool outlier_filter
int32 rad_outlier_min_neighbors
float32 outlier_filter_scale

# Plane RANSAC Segmentation
int32 max_iterations
float32 threshold_distance

# Raster Image
float32 pixel_wdt
float32 pixel_hgt

bool fill_holes
int32 hole_filling_neighbor_count

---

sensor_msgs/Image output_depth_image
sensor_msgs/Image output_intensity_image
sensor_msgs/PointCloud2 outlierless_cloud
sensor_msgs/PointCloud2 plane_cloud
sensor_msgs/PointCloud2 rotated_cloud
sensor_msgs/PointCloud2 transformed_cloud
sensor_msgs/PointCloud2 output_cloud
float32 image_fill_ratio
int32 image_wdt
int32 image_hgt
float32 depth_max
float32 depth_min
int32 intensity_min
int32 intensity_max
float32[] plane_coefficients